_people/felicity_chilver.md

```diff
@@ -6,5 +6,5 @@ award: Fellowship
 submitted: false
 graduated: 2018
 course:
-  - English Literature MA
+  - MA English Literature
```

Comment:
I assume there’s also a BA English that should be here?

Reply:
Not necessarily? I Thought if you did a straight 4 year masters then you skip the BA bit

Reply:
I guess there’s lots of nuance: sciences definitely have an integrated option, though Jess D did it separately (whereas David/Dan did not); and English is often optional (Callum has 2 degrees), but I suppose could be straight through.

Reply:
my thought iswe leave it as is unless she submits a bio which alters it?